test/built-ins/Array/prototype/withAt/frozen-this-value.js

```diff
@@ -1,18 +1,20 @@
+// |reftest| shell-option(--enable-change-array-by-copy) skip-if(!this.hasOwnProperty('getBuildConfiguration')||!getBuildConfiguration()['change-array-by-copy'])
 // Copyright (C) 2021 Igalia. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-esid: sec-array.prototype.with
+esid: sec-array.prototype.withAt
 description: >
-  Array.prototype.with works on frozen objects
+  Array.prototype.withAt works on frozen objects
```

Comment:
@acutmore is this the name we have consensus for?

Reply:
latest spec is `with`, hasn't been presented to plenary yet but I am hoping to get consensus for the current spec to stage 3.